sisyphus/manager.py

```diff
@@ -459,17 +458,22 @@ def startup(self):
 
         answer = None
         if gs.STATE_ERROR in self.jobs:
-            if self.clear_once:
-                self.clear_errors()
+            if self.clear_errors_once:
+                self.clear_states(state=gs.STATE_ERROR)
+                self.clear_errors_once = False
             elif self.ignore_once:
                 pass
             else:
                 answer = self.input('Clear jobs in error state? [y/N] ')
                 if answer.lower() == 'y':
-                    self.clear_once = True
-                    self.clear_errors()
+                    self.clear_states(state=gs.STATE_ERROR)
+                    self.clear_errors_once = False
                     self.print_state_overview(verbose=False)
                 answer = None
+        if gs.STATE_INTERRUPTED in self.jobs:
```

Comment:
This would be nicer in one line, but I'll merge it anyway.

Reply:
You're right, that was ugly. Fixed it now.